org.hlwd.sonofman.yml

```diff
@@ -0,0 +1,16 @@
+app-id: org.hlwd.sonofman
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: som
+finish-args:
+  - --filesystem=home
+modules:
+  - name: sonofman
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=${FLATPAK_DEST} --root=/ --optimize=1
```

Comment:
you probably need to list the python dependencies. See https://github.com/flatpak/flatpak-builder-tools that helpd for python.